library/default-synths-extra.scd

```diff
@@ -523,3 +523,146 @@ SynthDef(\superreese,  {|out, pan, freq, sustain, accelerate, detune=0, voice=0|
 	OffsetOut.ar(out, DirtPan.ar(sound, ~dirt.numChannels, pan, env));
 }).add;
 );
+
+(
+// DrumSynths SC Example - SOS Drums by Renick Bell, renick@gmail.com
+// recipes from Gordon Reid in his Sound on Sound articles
+// SOSkick — http:www.soundonsound.com/sos/jan02/articles/synthsecrets0102.asp
+// increase pitch1 and voice for interesting electronic percussion
+SynthDef(\soskick, {
+  arg out, pan, freq = 50, pitch1 = 5, voice = 5, sustain = 1, nl = 0.025;
```

Comment:
usually, we call filter frequency `ffreq`. Would that be a good name?

Reply:
Yeah, I think it will be fine, but then you cannot control frequency from `TidalCycles` (except `pF "ffreq" 200`, but it is quite unobvious).

What do you think about mapping frequency, passed from TidalCycles to only one octave for the kick? I mean this:
`d1 $ s "soskick" # freq 63` (or `c3`) -> stays unchanged and used "as is" 63Hz
`d1 $ s "soskick" # freq 130` (or `c4`) -> maps to 63Hz as well, cause it is out of defined limits

And the limits is the lower octave 63-123Hz

How about that behaviour? This Synthdef will always sound fine + you can tune it with TidalCycles as well!